src/main.jsx

```diff
@@ -1,15 +1,19 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
-import { RouterProvider } from "react-router-dom";
-import router from "./router/index.jsx";
-import "./index.scss";
-import mapboxgl from "mapbox-gl";
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { RouterProvider } from 'react-router-dom';
+import router from './router/index.jsx';
+import { Provider } from 'react-redux';
+import store from './store/redux/index.js';
+import './index.scss';
+import mapboxgl from 'mapbox-gl';
 
 mapboxgl.accessToken =
-  "pk.eyJ1IjoiY3Jpc3RpYW5qczkzIiwiYSI6ImNsa292YXBvcjBhcDIzcmw5cThnOGl6MTMifQ.K0CoYwD-yvtbr2VDTFecXQ";
+  'pk.eyJ1IjoiY3Jpc3RpYW5qczkzIiwiYSI6ImNsa292YXBvcjBhcDIzcmw5cThnOGl6MTMifQ.K0CoYwD-yvtbr2VDTFecXQ';
```

Comment:
this access token should be importable through a constant

Reply:
resolved